### DISCUSSION TITLE
IT SHOWS ME MY IDENTITY USED BEFORE AND WHEN I USED IT BEFORE IT ALWAYS REJECTED AND NOW IT SHOWS THIS AND STILL I NOT GOT STUDENT PACK

### DISCUSSION THREAD

**Discussion by AbhishekSachdeva07** [community/community#55201] [GitHub Education]
### Hi everyone,

IT SHOWS ME MY IDENTITY USED BEFORE AND WHEN I USED IT BEFORE IT ALWAYS REJECTED AND NOW IT SHOWS THIS AND STILL I NOT GOT STUDENT PACK

**Comment by samus-aran** [comment]
Hi @AbhishekSachdeva07 !

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
